### PR TITLE
Another minor nit in Python code

### PR DESCRIPTION
**Pull request by Luv2C0d3**
Same as previous diff.

### CODE REVIEW

File: part2/transports/disk/file_trans.py [hunk 1]
```diff
@@ -4,16 +4,13 @@
 from thrift.transport import TTransport
 
 class Trade:
-    def __init__(self):
-        symbol=""
-        price=0.0
-        size=0
+    def __init__(self, symbol, price, size):
```
Comment (RandyAbernethy) [inline part2/transports/disk/file_trans.py]: Think if we are going to pass them we should use them right? Pylint will squawk here.

**Comment by RandyAbernethy** [comment]
The init func from your last pull would be perfect here to.

File: part2/transports/disk/file_trans.py (hunk 1 above)
Comment (Luv2C0d3) [reply part2/transports/disk/file_trans.py]: I fat fingered. I will update it tomorrow morning.

Comment (Luv2C0d3) [reply part2/transports/disk/file_trans.py]: Ok, I checked now that indeed it does work:
```
$ python file_trans.py
Trade(86): F 2500 @ 13.100000
```

**Pull request merged**